app/jobs/expire_token_job.rb

```diff
@@ -0,0 +1,9 @@
+class ExpireTokenJob
+  include Sidekiq::Worker
```

Comment:
You can move `include` to `ApplicationJob` and then inherit from it. Would be cleaner

Reply:
`ArgumentError:
  You cannot include Sidekiq::Worker in an ActiveJob: ApplicationJob`